automate/tests/test_views.py

```diff
@@ -0,0 +1,98 @@
+from unittest.mock import patch
+
+from django.shortcuts import reverse
+
+from accounts.factories import UserFactory
+from automate.choices import RepoTypeChoices
+from automate.factories import ProjectFactory
+from automate.models import Project
+from repo.testing.api import BaseAPITestCase
+
+
+class ProjectAPITestCase(BaseAPITestCase):
+    """Test Project API Case."""
+
+    def setUp(self):
+        super().setUp()
+        self.user = UserFactory()
+        self.repo1 = ProjectFactory(owner=self.user)
+        self.data = {
+            "name": "Test Project",
+            "primary_repo_owner": "test-owner",
+            "primary_repo_name": "test-name",
+            "primary_repo_token": "11223344",
+            "primary_repo_type": RepoTypeChoices.GITHUB.value,
+            "secondary_repo_owner": "sec-owner",
+            "secondary_repo_name": "sec-name",
+            "secondary_repo_token": "2353w423",
+            "secondary_repo_type": RepoTypeChoices.BITBUCKET.value,
+        }
+        self.url_list = reverse("project:project-list")
+        self.url_detail = reverse(
+            "project:project-detail", kwargs={"slug": self.repo1.slug}
+        )
+
+    def test_to_ensure_only_authenticated_users_can_access_endpoint(self):
+        """Test to ensure user is logged in."""
+        with self.subTest("Accessing project-list"):
+            response = self.unauthenticated_client.get(self.url_list)
```

Comment:
This can simply be `self.client.get` since the client hasn't logged in yet.. It should still return 401